icu4j/main/classes/core/src/com/ibm/icu/impl/number/LongNameHandler.java

```diff
@@ -70,7 +73,12 @@ public PluralTableSink(String[] outArray) {
         public void put(UResource.Key key, UResource.Value value, boolean noFallback) {
             UResource.Table pluralsTable = value.getTable();
             for (int i = 0; pluralsTable.getKeyAndValue(i, key, value); ++i) {
-                int index = getIndex(key.toString());
+                String keyString = key.toString();
+                // TODO(ICU-21123): Load the correct inflected form, possibly from the "case" structure.
+                if (keyString.equals("case")) {
+                    continue;
+                }
```

Comment:
Busy working on case handling and making use of gender. This PR now gets ICU4J in sync with what's in master for ICU4C.

Reply:
+1